src/WebJobs.Script.WebHost/WebJobsScriptHostService.cs

```diff
@@ -156,32 +158,49 @@ private async Task StartHostAsync(CancellationToken cancellationToken, int attem
             }
             catch (OperationCanceledException)
             {
+                GetHostLogger(localHost).LogDebug("Host startup was canceled.");
```

Comment:
Can log this at info level? It might be worth it atleast initially to see these logs in Kusto to get better understanding for JobHost life cycle.

Reply:
Kusto sees debug logs. Writing debug is good for stuff like this bc customers won’t see it by default but we will.

Reply:
Thanks. Good to know!